sky/benchmark/benchmark_utils.py

```diff
@@ -114,8 +114,13 @@ def _print_candidate_resources(
                 f'{colorama.Style.RESET_ALL}')
 
     columns = [
-        'CLUSTER', 'CLOUD', '# NODES', 'INSTANCE', 'ACCELERATORS',
-        'PRICE ($/hr)'
+        'CLUSTER',
+        'CLOUD',
+        '# NODES',
+        'INSTANCE',
+        'vCPUS',
```

Comment:
`vCPUs` seems to be more [commonly used](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-optimize-cpu.html)

Reply:
Fixed. Thanks for making the poll!